api/staticRoutes.js

```diff
@@ -0,0 +1,69 @@
+const Router = require('express').Router();
+
+const Log = require('./lib/log');
+const Plan = require('./model/plan');
+const config = require('../src/config.json');
+
+// all plan pages should render opengraph tags
+Router.get(['/plan/:planId', '/plan/:planId/*'], function(req, res, next) {
+  const { planId } = req.params;
+
+  // fetch plan details to populate opengraph tags
+  Plan.fetchByPlanID(planId)
+    .then(plan => {
+      let pageDescription;
+
+      // some plans don't have depositing dates
+      if (plan.attributes.data.DEPOSITING_DATE)
+        pageDescription = `תב"ע ב${plan.attributes.data.PLAN_COUNTY_NAME} אשר הופקדה ב ${dateStrFormat(plan.attributes.data.DEPOSITING_DATE)}`;
+      else
+        pageDescription = `תב"ע ב${plan.attributes.data.PLAN_COUNTY_NAME}`;
+
+      res.render('index', {
+        layout: false, 
+        pageTitle: `${plan.attributes.PL_NUMBER} - ${plan.attributes.PL_NAME} - מעירים`, 
+        pageDescription: pageDescription,
+        pageImage: `${req.protocol}://${req.get('host')}/favicon.ico`,
+      });
+    }).catch(error => {
+      Log.error('failed to fetch plan details for opengraph');
+      Log.error(error);
+
+      res.render('index', {
+        layout: false, 
+        pageTitle: 'תוכנית בניה - מעירים', 
+        pageDescription: '',
+        pageImage: `${req.protocol}://${req.get('host')}/favicon.ico`,
+      });
+    });
+});
+  
+// all non-plan pages have static opengraph tag values
+Router.get('*', function(req, res, next) {
```

Comment:
We have some specific pages, let's define them so we can respond with 404 errors when links are broken. This is very bad for seo....